poc_config/openmrs/i18n/social/locale_en.json

```diff
@@ -5,5 +5,8 @@
   "RELATIONSHIP_TO_PATIENT": "Relationship to Patient",
   "HIV_STATUS": "HIV Status",
   "NO_PARTNERS": "No Sexual Partners.",
-  "PLEASE_CHECK_IN_TO_EDIT": "Please check-in the patient to be able to be able to edit."
+  "PLEASE_CHECK_IN_TO_EDIT": "Please check-in the patient to be able to be able to edit.",
+  "HIV INFECTED": "HIV+",
```

Comment:
@Guimino and @ynurmahomed I don't think this is the best solution. This is using the concept name as the i18n key but note that concept names can come in different languages. In my opinion the best approach is to change the concept name in the concept dictionary instead. By doing that we not only solve the i18n but also create naming consistency with all the other systems using the concept dictionary.